client/lib/post-normalizer/rule-create-better-excerpt.js

```diff
@@ -94,10 +91,11 @@ export default function createBetterExcerpt( post ) {
 	}
 
 	const strippedDom = buildStrippedDom( post.content );
-	post.content_no_html = striptags( strippedDom );
+
+	post.content_no_html = trim( striptags( strippedDom ) );
 
 	post.better_excerpt = formatExcerpt( strippedDom );
-	post.better_excerpt_no_html = trim( stripHTML( post.better_excerpt ) );
+	post.better_excerpt_no_html = trim( striptags( post.better_excerpt ) );
```

Comment:
hmm. I think we aren't even using `better_excerpt_no_html` anymore.
Maybe we can remove it in a future PR